src/routes/github/webhook/webhook-receiver-post.test.ts

```diff
@@ -132,4 +132,73 @@ describe("webhook-receiver-post", () => {
 		expect(GithubWebhookMiddleware).toBeCalledWith(issueWebhookHandler);
 	});
 
+	it("should call pull handler", async () => {
+		req = {
```

Comment:
So this is getting longer for each test. Can you see if you extract them into a method and reduce the clutter? 

```
req = {
			headers: {
				"x-hub-signature-256": "sha256=b3c1c0ce21666c0349b8f3010ce81299adcc9c4ee09b904f20f712292c19b792",
				"x-github-event": "pull_request"
			},
			params: {
				uuid
			},
			body: {
				action: "opened"
			}
		};
```

Reply:
@gxueatlassian , extracted in a seperate method